src/test/java/com/ksm/wstbackoffice/controller/CountryControllerIT.java

```diff
@@ -37,6 +40,23 @@
         );
     }
 
+    public static Stream<Arguments> badArgumentsProvider() {
+        return Stream.of(
+                Arguments.of(Arrays.asList(), 400),
+                Arguments.of(Arrays.asList(""), 400)
+        );
+    }
+
+    public static Stream<Arguments> correctArgumentsProvider() {
+        return Stream.of(
+                Arguments.of("amer", Arrays.asList("United States of America"), 200),
+                Arguments.of("kor", Arrays.asList("North Korea"), 200),
+                Arguments.of("sa", Arrays.asList("Saint Kitts and Nevis", "Saint Lucia",
```

Comment:
You provide only one filter parameter to the test. But our controller can accept the list of namesStartWithFilters. I want you to test this as well